common/jmeter/perf-test-is.sh

```diff
@@ -88,6 +88,7 @@ spCount=10
 idpCount=1
 userCount=1000
 mode=""
+use_db_snapshot="false"
```

Comment:
Can't we consider these as boolean without string?

Reply:
Fixed with https://github.com/wso2/performance-is/pull/200/commits/fca074e5c96788aae57ac5b8a969d57d916d2a91